### PR TITLE
updated readme to inform cloners to use strictMath less loader option

### PR DESCRIPTION
**Pull request by john1jan**
Just try this and check whether all your apps are working fine

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,5 +66,15 @@ Smaller bundles FTW!!!
 
 Let us know if you're using `preact-material-components` and love them. Send us a PR and add your projects and link right here.
 
+## Important!! less Users
```
Comment (prateekbh) [inline README.md]: Please change the heading to FAQ
add a section
>### Error while using `preact-material-components` with `less`
and then add the answer...

Thanks a lot for fixing this, its a huge contribution for us

File: README.md [hunk 2]
```diff
@@ -66,5 +66,15 @@ Smaller bundles FTW!!!
...
+- If you are using less-loader you have set **strictMath: true** in your webpack config
+```
+{
+	loader: 'less-loader',
```
Comment (prateekbh) [inline README.md]: spaces 😋

File: README.md [hunk 3]
```diff
@@ -66,5 +66,15 @@ Smaller bundles FTW!!!
...
+	options: { strictMath: true}
```
Comment (filoozom) [inline README.md]: Here one space too after ```strictMath: true```.

File: README.md (hunk 1 above)
Comment (filoozom) [reply README.md]: "Important!!" is maybe a little too dramatic for documentaion 😛

**Comment by john1jan** [comment]
formatted the readme. check it out

File: README.md [hunk 4]
```diff
@@ -66,5 +66,16 @@ Smaller bundles FTW!!!
 
 Let us know if you're using `preact-material-components` and love them. Send us a PR and add your projects and link right here.
 
+## FAQ
+### Error while using preact-material-components with **less**
+If you are using less-loader you have set **strictMath: true** in your webpack config
```
Comment (prateekbh) [inline README.md]: have to*

**Pull request merged**